python/ray/serve/_private/controller.py

```diff
@@ -1040,7 +1037,6 @@ class ServeControllerAvatar:
     def __init__(
```

Comment:
@edoakes @sihanwang41 Do you know where the `ServeControllerAvatar` is initialized? I couldn't find it.

Reply:
Java